server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/AnalyzeVisitor.java

```diff
@@ -160,7 +160,7 @@
 /** This visitor is used to analyze each type of Statement and returns the {@link Analysis}. */
 public class AnalyzeVisitor extends StatementVisitor<Analysis, MPPQueryContext> {
 
-  private static final Logger logger = LoggerFactory.getLogger(Analyzer.class);
```

Comment:
From my perspective, this bug fix very critical because this bug may cause many problems when we locate the problems. We need to highlight this type bug.